crates/noirc_evaluator/src/brillig/brillig_gen.rs

```diff
@@ -12,10 +12,6 @@ use self::{brillig_block::BrilligBlock, brillig_fn::FunctionContext};
 use super::brillig_ir::{artifact::BrilligArtifact, BrilligContext};
 
 /// Converting an SSA function into Brillig bytecode.
-///
-/// TODO: Change this to use `dfg.basic_blocks_iter` which will return an
```

Comment:
What was the conclusion to this question?

Reply:
When we implemented calls we checked that the reverse post order would explore all possible predecessor blocks before going into a block, so all registers that the function could have used up until that point were already known

Reply:
I think this was mostly about whether we can use `dfg.basic_blocks_iter` --  it was not clear if this was already doing rpo or if its doing post order (This is orthogonal to this PR and we can create an issue)

Reply:
Should I create an issue and leave the comment removed or do we want the comment there? However I think that the basic_blocks_iter just iterates in the order they were inserted into the dfg

Reply:
Yeah I think we can leave the comment removed and create an issue to specify the ordering for basic_blocks_iter in a doc comment above the function definition

Reply:
Ah I must have missed that before 🤔 -- no action needed :)